### ISSUE TITLE
Newsletter Display

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/tiamat-theme#1069]
There's a few issues with newsletters:

**Newsletter title**

The newsletter title should not display on the HTML email view

<img width="729" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/2ae62127-9547-4aad-a02c-5ad2005d1073">

**Special characters showing as HTML entity code in HTML email view**

<img width="729" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/7d89ef67-5b05-4801-b564-7d6f802852f4">

**Missing thumbnails**

Section content that includes media for thumbnails are not rendering on the full or email view modes.

Example: https://live-ucbprod-pts.pantheonsite.io/newsletter/infra-and-sustain-transportation-comm/september-2023-0

**Alignment**

 The content and various other page elements are not aligned together on the email display of newsletters. This shows at least 5 different positions compared to the 1 in the D7 version. Because of the missing thumbnails I can't tell if they are aligned or not.

D10 version:

<img width="705" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/6f98ad34-9773-4f2c-a869-72ead5138106">

D7 version:

<img width="812" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/3f993693-3c93-444c-9099-25b1aa7df372">

**Link Colors**

Links on dark backgrounds are blue instead of gold and don't have sufficient contrast. The example below shows only the link in the header, but footer links were also the wrong color. This is an issue on the classic, dark boxed, and simple newsletter themes.

<img width="682" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/39d538df-aaee-4d9d-a64f-7dea277dd5c9">

**Title and heading sizes**

Headings are much larger on the HTML email view which could lead to display problems on mobile devices. 

The D10 H2 measures 32px while the D7 H2 measures 20px.

**Content Title Links**

This is a migration issue:

It looks like content that did NOT have a link is getting a default value of <front> for their links, resulting in titles getting linked to the site homepage that were not intended to be links:

<img width="1397" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/474a0e4e-556c-4d19-8574-9217d8d20090">

<img width="968" alt="image" src="https://github.com/CuBoulder/tiamat-theme/assets/349713/a9477de0-4d6b-438e-bc74-2e85f8667a12">

**Comment by timurtripp** [comment]
Child issue: CuBoulder/migrate_express#43
